### PR TITLE
SWC-5786: bring back the spinner after the initial data load

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

File: src/demo/containers/playground/WidgetDemo.tsx [hunk 1]
```diff
@@ -100,13 +100,11 @@ export const WidgetDemo: React.FunctionComponent = () => {
       <br />
       <Checkbox
         label="Initially Checked Checkbox"
-        id="ch1"
```
Comment (jay-hodgson) [inline src/demo/containers/playground/WidgetDemo.tsx:103]: Attempt to fix GH action responsible for NPM publish, GH tag, and styleguidist output.  https://github.com/Sage-Bionetworks/Synapse-React-Client/runs/3604908922?check_suite_focus=true

Comment (nickgros) [reply src/demo/containers/playground/WidgetDemo.tsx:103]: This alone won't fix the automated deploy (there's other configuration we are missing), but I can take care of the rest.

File: src/lib/containers/user_profile_links/UserProjects.tsx [hunk 2]
```diff
@@ -72,7 +74,10 @@ export default function UserProjects({ userId }: UserProjectsProps) {
         </>
       )}
       {!isFetching && allRows.length == 0 && <div>Empty</div>}
-      {isFetching && <SkeletonTable numRows={5} numCols={1} />}
+      {isLoading && <SkeletonTable numRows={5} numCols={1} />}
+      {isFetching && !isLoading && <div className="placeholder">
+        <SynapseSpinner size={30} />
+      </div>}
```
Comment (nickgros) [inline src/lib/containers/user_profile_links/UserProjects.tsx:80]: Just my opinion: this spinner will pop up each time it re-fetches, and with our current config, React Query is very eager to refetch. I think for now we should omit this entirely (or we can consider changing our React Query config).

If we _need_ to tell the user that data is re/fetching, then I think we would want a more subtle indicator. We would probably want to talk to design about that.

Comment (jay-hodgson) [reply src/lib/containers/user_profile_links/UserProjects.tsx]: Works for me.  Spinners have been removed.

**Review by nickgros — APPROVED** [review]

**Pull request merged**